src/Components/Pages/dashboard.module.css

```diff
@@ -0,0 +1,130 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: rgb(67, 105, 176);
+}
+
+.navbar {
+  display: flex;
+  align-items: center;
+  background: rgb(67, 105, 177);
+  justify-content: space-between;
+  min-height: 4rem;
+  padding: 0.5rem;
+  color: white;
+}
+
+.brand {
+  margin-left: 40px;
+}
+
+.brand > h1 {
+  font-size: 20px;
+}
+
+.stocklist {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  color: white;
+  font-size: 17px;
+}
+
+.card {
+  background: rgb(62, 97, 163);
+  padding: 10px;
+  width: 50%;
+}
+
+.card:nth-child(2n) {
+  background-color: rgb(67, 105, 176);
+}
+
+.card:first-child {
```

Comment:
- [ ] Great job working on this this project and implementing the changes requested by the previous reviewer. However you have missed something which we have to fix. You have selected the blue theme from Behance which is very ok, but as the previous reviewer indicated, your cards color should follow an **alternating dual-color pattern**. You currently have **dual-color**, but it is not alternating. I will present your screenshot and that from Behance below for clarity.


| Your design | Original design |
| -------------  | ----------------- |
| ![Screen Shot 2023-02-07 at 10 08 08 PM](https://user-images.githubusercontent.com/62849872/217448759-7aa8861f-c9c8-4c75-a3f5-06e6e77bc427.png) | ![Screen Shot 2023-02-07 at 10 00 54 PM](https://user-images.githubusercontent.com/62849872/217447733-41dde8d2-3ed8-45a6-b2b5-a3e743d49b75.png)         |
Take a careful look at both screenshots, you will observe that in the original design the color on each column are not the same for the cells of that column. But from your own screenshot, you have a single color on the left column and a single color on the right colum. The colors should alternate for the cards of the same column.